schema/core.md

```diff
@@ -0,0 +1,193 @@
+# Core Metadata
+
+Core metadata is a set of common metadata fields that all catalog
+holdings share and consists largely of high-level dataset attributes
+and properties. This document outlines the **required** and **recommended**
+core metadata and includes example implementations for each.
+
+**Required Metadata**
+
+|Property|SchemaOrg Class|Expected Type|Cardinality|Description| 
+|---|---|---|---|---|
+|url|Thing|URL|1|the url of the item|
+|name|Thing|Text|1|the name of the item| 
+|description|Thing|Text|1|the description of the item|
+|author|Creative Work|Person OR Organization|1+| person or organization that authored the work|
+| dateCreated | CreativeWork | Date \| DateTime | 1 | The date on which the work was created|
+
+The following is an example implementation of the required metadata properties
+for a SchemaOrg Dataset item:
+
+``` json
+{
+  "@context": "https://schema.org/",
+  "@type": "Dataset",
+  "name": "IGUIDE Shapefile Testing Resource",
+  "description": "This resource contains medium-resolution (1:100k) National Hydrography Dataset (NHDPlus) [1] map data for a region of 39 Hydrologic Unit Code (HUC) 6-digit (HUC6) basins around the Hurricane Harvey impact zone across Texas, Louisiana, Mississippi and Arkansas. This includes 5978 subwatersheds, 190,192 catchments, and 192,267 flowlines. USGS active stream gages (924) were downloaded from the USGS National Water Information System (NWIS) [2] and augmented with each gage's HUC2, HUC4, HUC6, HUC8, HUC10 & HUC12 basin identifiers, and COMID of the NHD stream reach for the containing catchment. This allows the user to easily aggregate gages by various watershed boundaries. NOAA Advanced Hydrologic Prediction System (AHPS) [3] has 362 river forecast points in the Harvey study area. Many of these are co-located with USGS NWIS gages to leverage authoritative observation data. A shapefile of Texas dams (7290) was directly received from the Texas Commission for Environmental Quality (TCEQ) [4]. They suggest if you have any questions about data, to make an Open Records Request [5].",
+  "url": "https://www.hydroshare.org/resource/9d413b9d57824a79b8239a5f7c4fdf51/",
```

Comment:
I was under the impression that Identifiers should be unique and would therefore be created and maintained by the catalog, not within the resource-level metadata. We could allow data/resources to provide their own identifiers, however I imagine that we would largely ignore these. In many cases, the `url` element provides a unique identifier.

Reply:
I can see this being useful for a published work, but not necessarily required.

Reply:
I think it would be good for us to keep track of identifiers used by cataloged items - even if the catalog assigns them a unique identifier. I have seen a couple of search interfaces that provided a specific "find by identifier" function. In any case, those identifiers are the unique identifiers assigned by the system in which a cataloged item resides. And, it may be useful in disambiguation.